### ISSUE TITLE
Cannot train dreambooth because HFValidationError: Repo id Must use Alphanumeric Chars

### ISSUE THREAD

**Issue by Smushy22** [d8ahazard/sd_dreambooth_extension#626]
## Kindly read this ENTIRE form and fill it out with the requested information. It may seem tedious, but it's even more tedious asking basic questions to ensure I'm not trying to troubleshoot a bug I've already fixed. Help me help you, answer all the questions and follow the steps below before posting.

**Please find the following lines in the console and paste them below. _If you do not provide this information, your issue will be automatically closed._**

`
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: 98947d173e3f1667eba29c904f681047dea9de90
Installing requirements for Web UI
Skipping dreambooth installation.
Dreambooth revision is c5cb58328c555ac27679422b1da940a9b19de6f2
Diffusers version is 0.10.2
Torch version is 1.12.1+cu113
Torch vision version is 0.13.1+cu113
`

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes
**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes
**Have you updated Dreambooth to the latest revision?**
Yes
**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes
**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this. Reply 'OK' Below to acknowledge that you did this.**
OK
**Describe the bug**
I've been trying and failing to train a model, each time I click the train button it starts to initialize dreambooth until I receive the error "Exception training model: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'E:\Brian\stable-diffusion-webui\models\dreambooth\TwiBot\working\tokenizer'." then It stops. If I scroll up in the cmd console It tells me it was "huggingface_hub.utils._validators.HFValidationError" I've made sure that I accepted everything and that my email is verified, yet I still receive this problem
(A clear and concise description of what the bug is)

**Provide logs**
`Concept 0 class dir is E:\backup\pony
Custom model name is TwiBot
Starting Dreambooth training...
 Allocated 0.0/2.0GB
 Reserved: 0.0/2.0GB

Initializing dreambooth training...
Patching transformers to fix kwargs errors.
E:\Brian\stable-diffusion-webui\venv\lib\site-packages\transformers\generation_utils.py:24: FutureWarning: Importing `GenerationMixin` from `src/transformers/generation_utils.py` is deprecated and will be removed in Transformers v5. Import as `from transformers import GenerationMixin` instead.
  warnings.warn(
Replace CrossAttention.forward to use default
Checking concept: {'max_steps': -1, 'instance_data_dir': 'E:\\backup\\Poniue\\Twilighjt', 'class_data_dir': 'E:\\backup\\pony', 'instance_prompt': 'Digital art of TwiBot pony', 'class_prompt': 'Digital art of a pony', 'save_sample_prompt': '', 'save_sample_template': '', 'instance_token': '', 'class_token': '', 'num_class_images': 0, 'class_negative_prompt': '', 'class_guidance_scale': 7.5, 'class_infer_steps': 40, 'save_sample_negative_prompt': '', 'n_save_sample': 1, 'sample_seed': -1, 'save_guidance_scale': 7.5, 'save_infer_steps': 40}
Concept requires 0 images.
Traceback (most recent call last):
  File "E:\Brian\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\dreambooth.py", line 362, in start_training
    config, mem_record, msg = main(config, mem_record, use_subdir=use_subdir, lora_model=lora_model_name,
  File "E:\Brian\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\train_dreambooth.py", line 515, in main
    tokenizer = AutoTokenizer.from_pretrained(
  File "E:\Brian\stable-diffusion-webui\venv\lib\site-packages\transformers\models\auto\tokenization_auto.py", line 582, in from_pretrained
    tokenizer_config = get_tokenizer_config(pretrained_model_name_or_path, **kwargs)
  File "E:\Brian\stable-diffusion-webui\venv\lib\site-packages\transformers\models\auto\tokenization_auto.py", line 433, in get_tokenizer_config
    resolved_config_file = cached_file(
  File "E:\Brian\stable-diffusion-webui\venv\lib\site-packages\transformers\utils\hub.py", line 409, in cached_file
    resolved_file = hf_hub_download(
  File "E:\Brian\stable-diffusion-webui\venv\lib\site-packages\huggingface_hub\file_download.py", line 1022, in hf_hub_download
    cache_dir, repo_folder_name(repo_id=repo_id, repo_type=repo_type)
  File "E:\Brian\stable-diffusion-webui\venv\lib\site-packages\huggingface_hub\utils\_validators.py", line 92, in _inner_fn
    validate_repo_id(arg_value)
  File "E:\Brian\stable-diffusion-webui\venv\lib\site-packages\huggingface_hub\utils\_validators.py", line 142, in validate_repo_id
    raise HFValidationError(
huggingface_hub.utils._validators.HFValidationError: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'E:\Brian\stable-diffusion-webui\models\dreambooth\TwiBot\working\tokenizer'
 Training completed, reloading SD Model.
 Allocated: 0.0GB
 Reserved: 0.0GB

Memory output: {}
 Restored system models.
 Allocated: 2.0GB
 Reserved: 2.0GB

Returning result: Exception training model: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'E:\Brian\stable-diffusion-webui\models\dreambooth\TwiBot\working\tokenizer'.`
If a crash has occurred, please provide the *entire* stack trace from the log, including the last few log messages *before* the crash occurred.

**Environment**

What OS?
Windows 10 pro
If Windows - WSL or native?
Native
What GPU are you using?
Geforce RTX 3060
**Screenshots/Config**
If the issue is specific to an error while training, please provide a screenshot of training parameters or the db_config.json file from /models/dreambooth/MODELNAME/db_config.json
![image](https://user-images.githubusercontent.com/108608110/209508176-b54a68ca-9a78-453b-916b-e785455192ce.png)

**Comment by d8ahazard** [comment]
Please update and try with the latest version:

https://github.com/d8ahazard/sd_dreambooth_extension/commit/15244fef18c35864be6f6c2e1c0237dadc5415c1

**Comment by CD2902** [comment]
Just updated the latest version, and I'm also having issue.

**Comment by Flonixcorn** [comment]
same here also getting this error smhw

**Comment by TripleShahar** [comment]
same here

**Comment by Dizon78** [comment]
I got this error msg when I lost connection(session?) of Google Drive but still tried to 'Start Training'

At the 1st stage, of course I made, during going on, by any reason I lost the session.
(Maybe by taking time in updating codes, by changing Runtime type of CoLab, by managing sesseions. In don't know)
Re-visting from the begging, with Google Drive session re-built, then trying 'Start Training' again passed this error in my case.
